index.html

```diff
@@ -47,6 +47,11 @@ <h1>The Regis Altare Birthday Project 2023</h1>
         >
       </footer>
     </section>
+    <section id="card-list">
+      <!--Test image serving-->
+      <img class="card-image" src="images/elysium_delivery_services.jpg">
+      <tcg-card></tcg-card>
```

Comment:
Does this `<tcg-card></tcg-card>` line do anything? Delete if not?

Reply:
It's there to show we can use the component the same way as any other html tag (it's the red flower image), but we can delete it.

Reply:
ah, makes sense! Since you're the sole dev building on top of these changes and you know you'll remove it eventually, you can keep it for now